atdate/atdate_format.py

```diff
@@ -3,25 +3,30 @@
          | time date
          | time increment
          | time date increment
+         | isodate isotime
```

Comment:
Maybe it would be better to let users to use any kind of date and time format?
```suggestion
         | date time
```
Or we can make another rule `datetime`  to agregate time and date order.

Reply:
Yep, I thought about this, but wasn't sure. Your proposition with `date time` looks good enough IMO.